### PR TITLE
fix class use and in phpdoc

### PR DESCRIPTION
**Pull request by ewgRa**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

Just some fixes from code inspection.

### CODE REVIEW

File: src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/DoctrineExtensionTest.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ protected function setUp()
     }
 
     /**
-     * @expectedException LogicException
+     * @expectedException \LogicException
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/DoctrineExtensionTest.php]: expectedException always take a fqcn, which means there is no need for a starting `\` for these annotations

Comment (ewgRa) [inline src/Symfony/Bridge/Doctrine/Tests/DependencyInjection/DoctrineExtensionTest.php]: yes, but phpstorm can't understand this and show that this class is undefined. This is only phpstorm problem? In other IDE it works well with "Find usage" functionality for example, or when you click on this class to check its source?

**Comment by nicolas-grekas** [comment]
:+1: squashed+rebased on 2.7 (no need to update the PR we can do it when merging @ewgRa)

**Comment by Tobion** [comment]
:+1:

**Comment by Tobion** [comment]
Thank you @ewgRa.

**Pull request closed without merging**